pkg/gateway/server/router.go

```diff
@@ -62,6 +62,10 @@ func (s *Server) AddRoutes(mux *server.Server) {
 	mux.HandleFunc("GET /api/app-oauth/get-token", wrap(s.getTokenOAuthApp))
 	mux.HandleFunc("GET /api/app-oauth/get-token/{id}", wrap(s.getTokenOAuthApp))
 
+	// Handle updates to the file scanner configuration
+	mux.HandleFunc("GET /api/file-scanner-config", wrap(s.getFileScannerConfig))
+	mux.HandleFunc("PUT /api/file-scanner-config", wrap(s.updateFileScannerConfig))
```

Comment:
would you be able to delete?

Reply:
There is no reason to delete right now. Updating to set the values to empty strings has the same effect and we don't lose any updated/created information.